src/components/p5js-components/city.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import p5 from 'p5';
+import '../projects.css';
+
+
+class City extends React.Component {
+    constructor(props) {
+        super(props)
+        this.myRef = React.createRef()
+      }
+    Sketch = ( sketch ) => {
+
+        let xPos = 10;
+        let yPos = 10;
+        let xPlane = 0;
+        let yPlane = 50;
+        let width = 400;
+        let height = 400;
+        
+        sketch.setup = () => {
+          sketch.createCanvas(width, height);
+        };
+      
+        sketch.draw = () => {
+          sketch.background(29, 40, 115);
+      
+          sketch.fill(255, 242, 0);
+          sketch.ellipse(xPos, yPos, 10, 10);//star 1
+          sketch.ellipse(xPos+50, yPos, 10, 10);//star 2
+          sketch.ellipse(xPos+100, yPos, 10, 10);//star 3
+          sketch.ellipse(xPos+150, yPos, 10, 10);//star 4
+          sketch.ellipse(xPos+200, yPos, 10, 10);//star 5
+          //reset star movement
+          if (yPos >= height){
```

Comment:
```suggestion
          if (yPos >= height) {
```
consistency! if you're going to put a space before the bracket, do it everywhere. if you're not going to have a space, do it everywhere!